test/test_api/test_project_locks.py

```diff
@@ -0,0 +1,197 @@
+# -*- coding: utf8 -*-
+# This file is part of PYBOSSA.
+#
+# Copyright (C) 2015 Scifabric LTD.
+#
+# PYBOSSA is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PYBOSSA is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with PYBOSSA.  If not, see <http://www.gnu.org/licenses/>.
+import json
+from unittest.mock import patch, call
+
+from nose.tools import assert_equal
+
+from unittest.mock import patch
+
+from pybossa.model.project import Project
+from pybossa.api.project_locks import ProjectLocksAPI
+from test import db, with_context
+from test.factories import (ProjectFactory, UserFactory)
+from test.test_api import TestAPI
+
+
+class TestProjectAPI(TestAPI):
```

Comment:
Should this be `TestProjectLocksAPI`?

Reply:
Fixed in https://github.com/bloomberg/pybossa/pull/933/commits/86953d171ad5e6bd1a93c8e144d76862270f1484.